app/src/main/java/ch/epfl/skysync/database/schemas/FlightSchema.kt

```diff
@@ -25,33 +27,61 @@ data class FlightSchema(
-    /** We use the Date class instead of the LocalDate for Firestore see [DateLocalDateConverter] */
-    val date: Date? = null
+    /** We use the Date class instead of the LocalDate for Firestore see [DateUtility] */
+    val date: Date? = null,
+    /** In: Confirmed flight */
+    val remarks: List<String>? = null,
+    /** In: Confirmed flight */
+    val color: FlightColor? = null,
+    /** In: Confirmed flight */
+    val meetupTimeTeam: String? = null,
+    /** In: Confirmed flight */
+    val departureTimeTeam: String? = null,
+    /** In: Confirmed flight */
+    val meetupTimePassenger: String? = null,
+    /** In: Confirmed flight */
+    val meetupLocationPassenger: String? = null,
 ) : Schema<Flight> {
   override fun toModel(): Flight {
     throw NotImplementedError()
   }
 
   companion object {
     fun fromModel(model: Flight): FlightSchema {
-      val status =
-          when (model) {
-            is PlannedFlight -> FlightStatus.PLANNED
-            else ->
-                throw UnsupportedOperationException(
-                    "Unexpected class ${model.javaClass.simpleName}")
-          }
-      return FlightSchema(
-          id = model.id,
-          flightTypeId = model.flightType.id,
-          balloonId = model.balloon?.id,
-          basketId = model.basket?.id,
-          vehicleIds = model.vehicles.map { it.id },
-          status = status,
-          numPassengers = model.nPassengers,
-          timeSlot = model.timeSlot,
-          date = DateLocalDateConverter.localDateToDate(model.date),
-      )
+      return when (model) {
+        is PlannedFlight ->
+            FlightSchema(
+                id = model.id,
+                flightTypeId = model.flightType.id,
+                balloonId = model.balloon?.id,
+                basketId = model.basket?.id,
+                vehicleIds = model.vehicles.map { it.id },
+                status = FlightStatus.PLANNED,
+                numPassengers = model.nPassengers,
+                timeSlot = model.timeSlot,
+                date = DateUtility.localDateToDate(model.date),
+            )
+        is ConfirmedFlight ->
+            FlightSchema(
+                id = model.id,
+                flightTypeId = model.flightType.id,
+                balloonId = model.balloon?.id,
+                basketId = model.basket?.id,
+                vehicleIds = model.vehicles.map { it.id },
+                status = FlightStatus.CONFIRMED,
+                numPassengers = model.nPassengers,
+                timeSlot = model.timeSlot,
+                date = DateUtility.localDateToDate(model.date),
+                remarks = model.remarks,
+                color = model.color,
+                meetupTimeTeam = DateUtility.localTimeToString(model.meetupTimeTeam),
+                departureTimeTeam = DateUtility.localTimeToString(model.departureTimeTeam),
+                meetupTimePassenger = DateUtility.localTimeToString(model.meetupTimePassenger),
```

Comment:
Why are all these times `String` and not `Date ?

Reply:
For simplicity, I haven't had time to consult the documentation to see how time is handled.